source/usage-examples/insertOne.txt

```diff
@@ -25,8 +25,9 @@ can also pass a callback method as an optional third parameter.
 For more information on this method, see the
 :node-api-4.0:`insertOne() API documentation </classes/collection.html#insertone>`.
 
-If the operation creates a document, the ``insertedId`` field of this
-object is the ``_id`` of the inserted document. 
+If the operation creates a document, the object appends the ``_id``
+of the inserted document and references it as the objects ``insertedId``
+field.
```

Comment:
A few items here:
- The object does not do the appending. The operation, when successful, appends the field and value to the object.
- When using possessives, make sure to use apostrophe. E.g. "object's".
- Make sure to clearly identify to the reader that "creates a document" is the successful case and that it's referred to by the specific article ("the") rather than the unspecific one ("a"). Also, we might consider using "inserts" rather than "creates". E.g. "If the operation successfully inserts the document".